rules/bugfix.bzl

```diff
@@ -0,0 +1,23 @@
+def find_cc_toolchain(ctx):
+    """
+    Returns the current `CcToolchainInfo`.
+
+    Args:
+      ctx: The rule context for which to find a toolchain.
+
+    Returns:
+      A CcToolchainInfo.
+    """
+
+    # Check the incompatible flag for toolchain resolution.
+    if hasattr(cc_common, "is_cc_toolchain_resolution_enabled_do_not_use") and cc_common.is_cc_toolchain_resolution_enabled_do_not_use(ctx = ctx):
+        if "@rules_cc//cc:toolchain_type" in ctx.toolchains:
+            return ctx.toolchains["@rules_cc//cc:toolchain_type"]
+        fail("In order to use find_cc_toolchain, your rule has to depend on C++ toolchain. See find_cc_toolchain.bzl docs for details.")
```

Comment:
Presumably you could drop this fail, and make the if below an elif?

Reply:
The form of this function (the control-flow and returns) is identical to the internal version.

Once a version of bazel ships with a correct `find_cc_toolchain`, we should delete this and use the shipping version.